docs/emulator.md

```diff
@@ -17,22 +17,18 @@ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/
 make install
 ```
 
-- Add GPU capacity to the cluster using command
+-
```

Comment:
```suggestion
```

Reply:
Apart from this minor nit, change LGTM
/hold
/lgtm
/approve
Please feel free to unhold when you address the nit.